File: Lab1/t2.cpp
#include <stdio.h>
double abs(double num){
    if (num < 0)
        return -num;
    return num;
}
int main(){
    double number;
    printf("Enter a doubleing-point number: ");
    scanf("%lf", &number);
    printf("The absolute value of %lf is %lf.\n", number, abs(number));
    return 0;
}

File: Lab1/t2.kt
import java.util.*
import kotlin.math.abs

fun main(args: Array<String>){
    val num: Double
    val input = Scanner(System.`in`)
    print("Enter a floating-point number: ")
    num = input.nextDouble()
    println("The absolute value of $num is ${abs(num)}")
}